preferences.json

```diff
@@ -1,10 +1,10 @@
 {
   "guiSettings" : {
-    "windowWidth" : 1932.0,
-    "windowHeight" : 740.0,
+    "windowWidth" : 1294.6666259765625,
+    "windowHeight" : 694.6666870117188,
     "windowCoordinates" : {
-      "x" : 7,
-      "y" : 30
+      "x" : -6,
+      "y" : -6
     }
```

Comment:
This file shouldn't be pushed up right? Because the removed window width and height are set for 125% display

Reply:
hmm i dont recall touching this, im not sure why it changed. Anyway i chnaged to the value that is present in our upstream repo.

Reply:
it was me i accidentally pushed it up once earlier tdy, just change it to whatever seems user friendly enough